src/MessageBird/Client.php

```diff
@@ -212,4 +212,10 @@ private function getPhpVersion()
 
         return 'PHP/' . PHP_VERSION_ID;
     }
+
+    public function addHttpOption($key, $value) {
```

Comment:
Having this function in `Client` leaks implementation details of `HttpClient`, so please remove this. Having `addHttpOption` in the HTTP client is sufficient